Update cards-format.md (#2913)
MicrosoftDocs/msteams-docs: msteams-platform/task-modules-and-cards/cards/cards-format.md

### Information masking in Adaptive cards

> Use the information masking property to mask specific information, such as password or sensitive information from users within the adaptive card [Input Text](https://adaptivecards.io/explorer/Input.Text.html) input element. Also note that feature only supports client side information masking, the masked input text is sent as clear text to the https endpoint address that was specified during [bot configuration](https://docs.microsoft.com/microsoftteams/platform/build-your-first-app/build-bot#4-configure-your-bot). 

Reviewer: ```suggestion
Use the information masking property to mask specific information, such as password or sensitive information from users within the Adaptive card [`Input.Text`](https://adaptivecards.io/explorer/Input.Text.html) input element. 

> [!NOTE]
> The feature only supports client side information masking, the masked input text is sent as clear text to the https endpoint address that was specified during [bot configuration](../../build-your-first-app/build-bot#4-configure-your-bot). 
```

If you have not set the `width` property to *Full*, then the default view of the Adaptive Card is as follows:
![Small width Adaptive Card view](../../assets/images/cards/small-width-adaptive-card.png)

> ### Typeahead support
> Within the [Input Choiceset](https://adaptivecards.io/explorer/Input.ChoiceSet.html) schema element, asking users to filter through and select through a sizable number of choices can signnificantly slow down task completion. Typeahead support within adaptive cards can simplfy input selection by narrowing (filtering) the set of input choices as a users is typing input. 

Reviewer: ```suggestion
### Typeahead support

Within the [`Input.Choiceset`](https://adaptivecards.io/explorer/Input.ChoiceSet.html) schema element, asking users to filter through and select through a sizable number of choices can significantly slow down task completion. Typeahead support within Adaptive cards can simplify input selection by narrowing or filtering the set of input choices as a user is typing the input. 
```

> #### Enable typeahaead in adaptive cards
> To enable typeahead within the Input choiceset set `style` to `filtered` and ensure `isMultiSelect` is set to `false`. 

Reviewer: ```suggestion
#### Enable typeahead in Adaptive cards

To enable typeahead within the `Input.Choiceset` set `style` to `filtered` and ensure `isMultiSelect` is set to `false`. 
```
